### ISSUE TITLE
"TypeError: Cannot read property 'find' of undefined" with Babel 7.0.0-beta.2.

### ISSUE THREAD

**Issue by phyllisstein** [tleunen/eslint-import-resolver-babel-module#67]
Running the latest beta of Babel 7, I'm seeing some trouble with the reference to `OptionManager.memoisedPlugins`:

```
TypeError: Cannot read property 'find' of undefined
    at /Users/daniel/Repos/Bauer/penelope/node_modules/eslint-import-resolver-babel-module/lib/index.js:19:47
    at Array.filter (<anonymous>)
    at getPlugins (/Users/daniel/Repos/Bauer/penelope/node_modules/eslint-import-resolver-babel-module/lib/index.js:18:27)
    at Object.exports.resolve (/Users/daniel/Repos/Bauer/penelope/node_modules/eslint-import-resolver-babel-module/lib/index.js:71:21)
    at v2 (/Users/daniel/Repos/Bauer/penelope/node_modules/eslint-module-utils/resolve.js:79:23)
    at withResolver (/Users/daniel/Repos/Bauer/penelope/node_modules/eslint-module-utils/resolve.js:84:16)
    at fullResolve (/Users/daniel/Repos/Bauer/penelope/node_modules/eslint-module-utils/resolve.js:101:22)
    at relative (/Users/daniel/Repos/Bauer/penelope/node_modules/eslint-module-utils/resolve.js:46:10)
    at resolve (/Users/daniel/Repos/Bauer/penelope/node_modules/eslint-module-utils/resolve.js:172:12)
    at resolveImportType (/Users/daniel/Repos/Bauer/penelope/node_modules/eslint-plugin-import/lib/core/importType.js:74:65)
```

Seems to happen in both a Lerna-managed repo and a standard-issue flat repo. Unfortunately the Babel team moved the location of the `OptionManager` class in Git so I'm not sure how to figure out when the property might have changed or what it might have changed to. If you have any thoughts I'd be glad to hear them!

**Comment by tleunen** [comment]
Thank @phyllisstein. I'll be working with adding compatibility with Babel 7 soon.

**Comment by tleunen** [comment]
Fixed in master, with the help of @nodkz.
I'll make a release tomorrow or the day after depending of my time. You can also expect to see it as the real v4. No more beta.

**Comment by phyllisstein** [comment]
That's awesome! Thank you so much, I appreciate the fast turnaround and all your hard work on this package.

**Comment by dmk255** [comment]
@tleunen - Was this error solved? I am running v4 and am receiving the same error as above.

```
settings:
  import/resolver:
    babel-module:
      extensions:
        - ".js"
        - ".jsx"
```

Note, I have tried this with both a `.babelrc` and a `.babelrc.js`.

Additionally, I have the following repository structure (from lerna). I don't think it should impact this plugin though. 
<details>
```
packages
-> package_1
-> -> .babelrc
-> -> package.json
-> -> .eslintrc.yml // where I implement the above config
-> -> src/...
.eslintrc.yml          // base eslint config
.babelrc
package.json
```
</details>
 
My babel config is:
<details>

```JSON
{
  "presets": [
      [
          "@babel/preset-env",
          {
              "shippedProposals": true,
              "modules": false,
              "comments": true,
              "targets": {
                  "browsers": [
                      "last 2 versions"
                  ]
              }
          }
      ],
      "@babel/preset-react"
  ],
  "plugins": [
      "@babel/plugin-proposal-class-properties",
      "syntax-dynamic-import",
      [
          "styled-components",
          {
              "displayName": true
          }
      ],
      [
          "module-resolver",
          {
              "root": [
                  "./src"
              ],
              "alias": {
                  "^Components(.*)$": "./src/Components\\1",
                  "^Constants(.*)$": "./src/Constants\\1",
                  "^Assets(.*)$": "./src/Assets\\1",
                  "^Utils(.*)$": "./src/Utils\\1"
              }
          }
      ]
  ],
  "env": {
      "test": {
          "presets": [
              [
                  "@babel/preset-env",
                  {
                      "shippedProposals": true,
                      "comments": true,
                      "targets": {
                          "browsers": [
                              "last 2 versions"
                          ]
                      }
                  }
              ],
              "@babel/preset-react"
          ],
          "plugins": [
              "@babel/plugin-proposal-class-properties",
              "syntax-dynamic-import",
              [
                  "styled-components",
                  {
                      "displayName": true
                  }
              ],
              [
                  "module-resolver",
                  {
                      "root": [
                          "./src"
                      ],
                      "alias": {
                          "^Components(.*)$": "./src/Components\\1",
                          "^Constants(.*)$": "./src/Constants\\1",
                          "^Assets(.*)$": "./src/Assets\\1",
                          "^Utils(.*)$": "./src/Utils\\1"
                      }
                  }
              ]
          ]
      }
  }
}
```
</details>

**Comment by Tomekmularczyk** [comment]
I'm using NextJS with `"@babel/core": "7.0.0-beta.42",` and I get the same error...

**Comment by klujanrosas** [comment]
@Tomekmularczyk I upgraded to `^5.0.0-beta.0` and the error went away, using next 6 stable

**Comment by trevordmiller** [comment]
I had same error using Next.js 6.0 (uses Babel 7) and it was also fixed by upgrading from `^4.0.0` to `^5.0.0-beta.0` as @klujanrosas did.
